tests/benchmarks/multicore/idle_spim_loopback/testcase.yaml

```diff
@@ -0,0 +1,83 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore spim
```

Comment:
Test that shall have PPK connected, override this field with `tags: ppk_power_measure`. See line 24, 36, 61, 74.